types/rdf-utils-dataset/resource.d.ts

```diff
@@ -0,0 +1,3 @@
+import { resource } from '.';
+
+export = resource;
```

Comment:
Oh, curious how you reversed the actual exports 🤔

Reply:
Just copied the pattern from other DT libs...

Reply:
That they re-export in named module rather the in the index?

Reply:
Was trying to mirror `rdf-dataset-ext`'s types as they look the same style.